gradle.properties

```diff
@@ -1,20 +1,17 @@
+projectVersion=6.0.0-SNAPSHOT
+
+asciidoctorJvmVersion=4.0.3
 grailsVersion=7.0.0-SNAPSHOT
 grailsGradlePluginVersion=7.0.0-SNAPSHOT
-groovyVersion=4.0.22
-liquibaseHibernate5Version=4.27.0
-projectVersion=6.0.0-SNAPSHOT
-hibernate5Version=5.6.15.Final
-servletApiVersion=6.0.0
+groovyVersion=4.0.23
```

Comment:
`groovyVersion` is not used (and `groovy-bom` is also imported in the `grails-bom`)

Reply:
It is not referenced directly, but is used when org.grails:grails-gradle-plugin is applied.

Reply:
Do you mean here:
https://github.com/grails/grails-gradle-plugin/blob/755b810a1fc0b8c9a2b6774f0816635ac6d9afc7/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy#L262

Do we still need that applied. If `groovyVersion` is `null` it does not do anything.

Reply:
That is the correct location.  Placing it there allowed us to remove the dependency substitution code through the projects, as long as they use grails-gradle-plugin.  There was a version of this code in grails-gradle-plugin:6.0.x.

It is probably still needed for a few edge cases.

Reply:
Ok 👍